sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -655,41 +657,63 @@ select interval (-30) day
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.sql.catalyst.parser.ParseException
+org.apache.spark.sql.AnalysisException
+Undefined function: 'interval'. This function is neither a registered temporary function nor a permanent function registered in the database 'default'.; line 1 pos 7
```

Comment:
ditto, it's better to make `day` and `days` have consistent behavior/error message.